ui_framework/src/components/popover/popover_title.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const KuiPopoverTitle = ({ children, className, ...rest }) => {
+
+  const classes = classNames(
+    {
+      'kuiPopoverTitle': children,
```

Comment:
`children` isn't clear here. It needs a test, plus we should maybe change this to `children != null` or something like that? It's difficult to understand the intention right now

Reply:
Great catch... I don't think this is necessary at all.